app/models/user.rb

```diff
@@ -5,6 +5,9 @@ class User < ActiveRecord::Base
   require 'net/http'
   require 'digest/md5'
 
+  enum dashboard_sort:   [:sort_by_date, :sort_by_group]
+  enum dashboard_filter: [:show_all, :show_unread]
```

Comment:
How are these enums feeling for you? I have a hunch that when we're not doing scales.. it's easier to read, debug, write as plain strings